native/general.go

```diff
@@ -33,3 +76,30 @@ func max(a, b int) int {
 	}
 	return b
 }
+
+func sign(a, b float64) float64 {
```

Comment:
Done.

Do you anything about the history of this function? It's useful here, but it seems like a weird thing to have in the math package (and yet a bunch of languages seem to have it easily accessible). Is this a lot more useful than I think?